File: summer/WebContent/js/system/windows/data/IValueConverter.js
/**
 * IValueConverter
 */

define(["dojo/_base/declare", "system/Type"], 
		function(declare, Type){
	var IValueConverter = declare("IValueConverter", Object,{
	});
	
	IValueConverter.Type = new Type("IValueConverter", IValueConverter, [Object.Type]);
	return IValueConverter;
});
//public interface IValueConverter
//	{
//		object Convert(object value, Type targetType, object parameter, CultureInfo culture);
//		object ConvertBack(object value, Type targetType, object parameter, CultureInfo culture);
//	}

File: org.summer.view.widget/src/org/summer/view/widget/data/IValueConverter.java
package org.summer.view.widget.data;

import org.summer.view.widget.CultureInfo;
import org.summer.view.widget.Type;

public interface IValueConverter {
	Object Convert(Object value, Type targetType, Object parameter, CultureInfo culture);

	Object ConvertBack(Object value, Type sourceType, Object parameter, CultureInfo culture) ;
}